Link to DJM from Databricks Documentation (#18125)
DataDog/integrations-core: databricks/README.md

# Agent Check: Databricks

> <div class="alert alert-warning">Discover our new <a href="https://docs.datadoghq.com/data_jobs/">Data Jobs Monitoring</a> product for observing, troubleshooting, and cost-optimizing your Spark and Databricks jobs and clusters.

Reviewer: ```suggestion
<div class="alert alert-warning"><a href="https://docs.datadoghq.com/data_jobs/">Data Jobs Monitoring</a> helps you observe, troubleshoot, and cost-optimize your Spark and Databricks jobs and clusters.<br/><br/>
```

just removing anything that sounds like marketing
